designs/administrator/olm/global-operator-all-namespace-operands/index.md

```diff
@@ -0,0 +1,41 @@
+---
+parent: Administrator
+version: 4.7
+---
+
+# Display All Namespace Operands for Global Operators
+
+When viewing the Installed Operators list set to 'All projects' and then selecting an operator that is available in 'All namespaces' (globally installed,) upon clicking the operator to view its details the user is taken into the details of that operator in installed namespace (the project selector will switch to the install namespace.)
```

Comment:
What if the description said something like:

Today in the console, when a user clicks on an operator that is available in “All namespaces” (globally installed) from the “All Projects” perspective on the Installed Operators page, they are taken to a the details page of where that particular operator is installed. The project selector switches from "All projects" to the specific project name where the operator was installed.

Reply:
I included this clearer text, thanks a lot for suggesting that.